src/types/symbol.ts

```diff
@@ -0,0 +1,21 @@
+import { Runtype, create, validationError } from '../runtype';
+
+export const __global = Function('return this')();
```

Comment:
I don't really understand the need for this. If the goal is to avoid clashing with the `Symbol` class, just use renaming exports/imports as we do with `Array`.

Reply:
Ah sorry, that was just a temporary workaround while I was developing.